### ISSUE TITLE
split sync folder from webdav path

### ISSUE THREAD

**Issue by hurradieweltgehtunter** [hurradieweltgehtunter/wordpress-owncloud-integration#12]
atm the whole webdav sync path gets stored.
Split this to save only the variable part (the actual folder sync path)
`remote.php/webdav/` can be hardcoded as it is static.

**Comment by yannik-b** [comment]
No, it is not static because you can install an OC installation into subfolders like example.com/cloud, example.com/owncloud, example.com/mycloud. Hardcoded `remote.php` won't work for those installations.

**Comment by hurradieweltgehtunter** [comment]
i think we're misunderstanding each other.
have a look :https://github.com/hurradieweltgehtunter/wordpress-owncloud-integration/blob/447b9529a764e95182d9771d0faeadb9065d6a56/pluginpage.php#L36

This is a very rough approach but it works. 
In my understanding `remote.php/webdav/` is a static webdav resource path. 
I tested it with my local ownCloud on different urls: localhost/owncloud, localhost/myowncloud
I'm sure one could improve the stuff I did there but for now it works.
